packages/terra-form/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Unreleased
 ### Added
 * Add missing props-table
 
+### Breaking Changes
+
+* Select and SelectField no longer accepts a list of strings, but a list of objects to set their value/display
```

Comment:
Updated to have an `options` prop, with `choices` listed as a deprecated prop that logs a warning message if used.